demos/motion_tracking/process_out/process.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+# See: https://spdx.org/licenses/
+
+from lava.magma.core.process.process import AbstractProcess
+from lava.magma.core.process.ports.ports import InPort
+from lava.magma.core.sync.protocols.loihi_protocol import LoihiProtocol
+from lava.magma.core.model.py.ports import PyInPort
+from lava.magma.core.model.py.type import LavaPyType
+from lava.magma.core.resources import CPU
+from lava.magma.core.decorator import implements, requires
+from lava.magma.core.model.py.model import PyLoihiProcessModel
+import numpy as np
+
+
+class ProcessOut(AbstractProcess):
+    """Process that receives (1) the raw DVS events, (2) the spike rates
+    of the selective as well as (3) the multi-peak DNF per pixel. It sends
+    these values through a multiprocessing pipe (rather than a Lava OutPort)
+    to allow for plotting."
+    """
+    def __init__(self,
+                 shape_dvs_frame,
+                 shape_dnf,
+                 send_pipe):
+        super().__init__(shape_dvs_frame=shape_dvs_frame,
+                         shape_dnf=shape_dnf,
+                         send_pipe=send_pipe)
+        self.dvs_frame_port = InPort(shape=shape_dvs_frame)
+        self.dnf_multipeak_rates_port = InPort(shape=shape_dnf)
+        self.dnf_selective_rates_port = InPort(shape=shape_dnf)
+
+
+@implements(proc=ProcessOut, protocol=LoihiProtocol)
+@requires(CPU)
+class DataRelayerPM(PyLoihiProcessModel):
```

Comment:
Why is this called `DataRelayerPM` when the Process is called `ProcessOut`?

Reply:
Renamed it to ProcessOutModel